config.json

```diff
@@ -1,4 +0,0 @@
-{
```

Comment:
I agree that `config.json` probably shouldn't be checked in, but maybe we can check in a file like `config.sample.json` w/ example parameters?